### PR TITLE
docs: improve clarity - oidc client field is required

### PR DESCRIPTION
**Pull request by Crowley723**
Improve documentation clarity by adding "required" tag to OIDC provider field: clients.

It is intended behavior that there be at least one configured open id connect client in order for OIDC config to be valid.

### CODE REVIEW

File: docs/content/configuration/identity-providers/openid-connect/provider.md [hunk 1]
```diff
@@ -553,6 +551,7 @@ Automatically adds the origin portion of all redirect URI's on all clients to th
 localhost.
 
 ### clients
+{{< confkey type="list(object)" required="yes" >}}
```
Comment (james-d-elliott) [inline docs/content/configuration/identity-providers/openid-connect/provider.md:554]:
```suggestion

{{< confkey type="list(object)" required="yes" >}}
```

**Review by james-d-elliott — CHANGES_REQUESTED** [review]
LGTM, one commented change. Also revert the unrelated changes.

File: docs/content/configuration/identity-providers/openid-connect/provider.md (hunk 1 above)
Comment (Crowley723) [reply docs/content/configuration/identity-providers/openid-connect/provider.md:554]: I swear there was only the tag change. Apologies.

Comment (james-d-elliott) [reply docs/content/configuration/identity-providers/openid-connect/provider.md:557]: Nah all good, that's what review is for.

**Review by james-d-elliott — APPROVED** [review]

**Pull request merged**